src/ZPublisher/WSGIPublisher.py

```diff
@@ -201,7 +202,9 @@ def _publish_response(request, response, module_info, _publish=publish):
             response.setBody(view())
             return response
 
-        raise exc
+        # Reraise the exception, preserving the original traceback
+        t, v, tb = sys.exc_info()
```

Comment:
I guess you mean because of this issue: https://cosmicpercolator.com/2016/01/13/exception-leaks-in-python-2-and-3/

Thanks for catching that; will fix.